deployment/GI03FLMB_Deploy.csv

```diff
@@ -1,118 +1,118 @@
+AR7-01,,AR21,,GI03FLMB-RIM01-02-CTDMOG065,3,1,2016-07-13T15:49:00,2017-08-11T10:15:00,CGMGI-03FLMB-00003,,CGINS-CTDMOG-11606,59.71547,-39.31479,,180,2825,
+AR7-01,,AR21,,GI03FLMB-RIM01-02-CTDMOG066,3,1,2016-07-13T15:49:00,2017-08-11T10:15:00,CGMGI-03FLMB-00003,,CGINS-CTDMOG-11603,59.71547,-39.31479,,250,2825,
+AR7-01,,AR21,,GI03FLMB-RIM01-02-CTDMOG067,3,1,2016-07-13T15:49:00,2017-08-11T10:15:00,CGMGI-03FLMB-00003,,CGINS-CTDMOG-11594,59.71547,-39.31479,,350,2825,
+AR7-01,,AR21,,GI03FLMB-RIM01-02-CTDMOG068,3,1,2016-07-13T15:49:00,2017-08-11T12:02:00,CGMGI-03FLMB-00003,,CGINS-CTDMOG-11645,59.71547,-39.31479,,500,2825,
+AR7-01,,AR21,,GI03FLMB-RIM01-02-CTDMOH069,3,1,2016-07-13T15:49:00,2017-08-11T12:02:00,CGMGI-03FLMB-00003,,CGINS-CTDMOH-12222,59.71547,-39.31479,,750,2825,
+AR7-01,,AR21,,GI03FLMB-RIM01-02-CTDMOH070,3,1,2016-07-13T15:49:00,2017-08-11T12:02:00,CGMGI-03FLMB-00003,,CGINS-CTDMOH-12223,59.71547,-39.31479,,1000,2825,
+AR7-01,,AR21,,GI03FLMB-RIM01-02-CTDMOH071,3,1,2016-07-13T15:49:00,2017-08-11T12:02:00,CGMGI-03FLMB-00003,,CGINS-CTDMOH-12226,59.71547,-39.31479,,1500,2825,
+AR7-01,,AR21,,GI03FLMB-RIS01-00-SIOENG000,3,1,2016-07-13T15:49:00,2017-08-11T10:15:00,CGMGI-03FLMB-00003,,CGCON-SIOCON-10118,59.71547,-39.31479,,30,2825,
+AR7-01,,AR21,,GI03FLMB-RIS01-03-DOSTAD000,3,1,2016-07-13T15:49:00,2017-08-11T10:15:00,CGMGI-03FLMB-00003,,CGINS-DOSTAD-00393,59.71547,-39.31479,,30,2825,
+AR7-01,,AR21,,GI03FLMB-RIS01-04-PHSENF000,3,1,2016-07-13T15:49:00,2017-08-11T10:15:00,CGMGI-03FLMB-00003,,CGINS-PHSENF-P0104,59.71547,-39.31479,,30,2825,
+AR7-01,,AR21,,GI03FLMB-RIS01-05-FLORTD000,3,1,2016-07-13T15:49:00,2017-08-11T10:15:00,CGMGI-03FLMB-00003,,CGINS-FLORTD-01122,59.71547,-39.31479,,30,2825,
+AR21,,AR30-03,,GI03FLMB-RI000-00-CTDMOH000,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-CTDMOH-13454,59.7178,-39.3525,,1812,2817,
+AR21,,AR30-03,,GI03FLMB-RI000-00-CTDMOH100,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-CTDMOH-13457,59.7178,-39.3525,,2712,2817,
+AR21,,AR30-03,,GI03FLMB-RI000-00-CTDMOH400,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-CTDMOH-13456,59.7178,-39.3525,,2412,2817,
+AR21,,AR30-03,,GI03FLMB-RI000-00-CTDMOH700,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-CTDMOH-13455,59.7178,-39.3525,,2112,2817,
+AR21,,AR30-03,,GI03FLMB-RI000-00-VELPTB000,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-VELPTB-12095,59.7178,-39.3525,,1812,2817,
+AR21,,AR30-03,,GI03FLMB-RI000-00-VELPTB100,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-VELPTB-12673,59.7178,-39.3525,,2712,2817,
+AR21,,AR30-03,,GI03FLMB-RI000-00-VELPTB400,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-VELPTB-12459,59.7178,-39.3525,,2412,2817,
+AR21,,AR30-03,,GI03FLMB-RI000-00-VELPTB700,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-VELPTB-12197,59.7178,-39.3525,,2112,2817,

```

Comment:
The OSNAP depths are incorrect (as is the spreadsheet - my bad) they should:
1812 –> 1817
2712 –> 2717
2412 –> 2417
2112 –> 2117

Reply:
corrected